File: dongwoo/week2/easy.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] rrr = {"aya", "ye", "woo", "ma"};
        for (String s : babbling) {
            if (s.contains("ayaaya") || s.contains("yeye") || s.contains("woowoo") || s.contains("mama")) {
                continue;
            }

            for (String r : rrr) {
                s = s.replace(r, " ");
            }

            if (s.trim().length() == 0) {
                answer++;
            }
        }
        return answer;
    }
}

File: juha/week5/easy.js
function solution(n) {
  const a = 1234567;

  let f = [0, 1];
  for (let i = 2; i <= n; i++) {
    f[i] = (f[i - 1] + f[i - 2]) % a;
  }
  return f[n];
}
